flash_examples/image_classification.py

```diff
@@ -29,7 +29,7 @@
 model = ImageClassifier(backbone="resnet18", num_classes=datamodule.num_classes)
 
 # 3. Create the trainer and finetune the model
-trainer = flash.Trainer(max_epochs=3, gpus=torch.cuda.device_count())
+trainer = flash.Trainer(max_epochs=3, gpus=torch.cuda.device_count(), limit_train_batches=2, limit_val_batches=2)
```

Comment:
Looks like this was an accident:
```suggestion
trainer = flash.Trainer(max_epochs=3, gpus=torch.cuda.device_count())
```